tests/integration_tests/test_filters.py

```diff
@@ -15,6 +15,8 @@
     w3_wait_for_new_blocks,
 )
 
+pytestmark = pytest.mark.filter
```

Comment:
@dudong2 
Q. Could you explain what this line of code does?

Reply:
Integration test is splited to 3 parts(filter, upgrade, unmarked).
The line means that this test file is belonged to filter test part.
If you test only filter part, you can try with this cmd(`make run-integration-tests TESTS_TO_RUN=filter`)